versioned_docs/version-v1.4/terraform/terraform-provider.md

```diff
@@ -13,6 +13,11 @@ title: "Harvester Terraform Provider"
 
 | Harvester Version                                                    | Supported Terraform Provider Harvester                                                  | Supported Terraformer Harvester                                                            |
 |----------------------------------------------------------------------|-----------------------------------------------------------------------------------------| ------------------------------------------------------------------------------------------ |
+| [v1.4.2](https://github.com/harvester/harvester/releases/tag/v1.4.2) | [v0.6.7](https://github.com/harvester/terraform-provider-harvester/releases/tag/v0.6.7) | [v1.1.1-harvester](https://github.com/harvester/terraformer/releases/tag/v1.1.1-harvester) |
+| [v1.4.1](https://github.com/harvester/harvester/releases/tag/v1.4.1) | [v0.6.7](https://github.com/harvester/terraform-provider-harvester/releases/tag/v0.6.7) | [v1.1.1-harvester](https://github.com/harvester/terraformer/releases/tag/v1.1.1-harvester) |
+| [v1.4.0](https://github.com/harvester/harvester/releases/tag/v1.4.0) | [v0.6.7](https://github.com/harvester/terraform-provider-harvester/releases/tag/v0.6.7) | [v1.1.1-harvester](https://github.com/harvester/terraformer/releases/tag/v1.1.1-harvester) |
```

Comment:
Can we have multiple versions support for 1.4.? We have tested 0.6.5 & 0.6.6 and they worked and same has been updated in the support matrix.
<img width="787" alt="Screenshot 2025-04-11 at 12 17 50 PM" src="https://github.com/user-attachments/assets/57e5cca5-38af-441a-ad0e-8fb36db2d2c2" />

Reply:
Updated it. Thanks.